torch/csrc/jit/script/script_type_parser.cpp

```diff
@@ -332,6 +332,28 @@ FunctionSchema ScriptTypeParser::parseSchemaFromDef(
       name, "", std::move(args), std::move(returns), false, false);
 }
 
+c10::IValue ScriptTypeParser::parseClassConstant(const Assign& assign) {
+  TORCH_INTERNAL_ASSERT(assign.lhs().kind() == TK_VAR);
```

Comment:
Please `throw ErrorReport` instead of `TORCH_INTERNAL_ASSERT`. People maybe editing the serialized code manually and it would be confusing to see these shown as internal asserts